lib/modules/security/index.js

```diff
@@ -194,75 +164,62 @@ Security.prototype.__validateRequest = function (methodName, callArguments, call
 };
 
 this.__cachedSystemPermissions = null;
-
 //Return possible assignable system permissions by iterating through the mesh description
-
-Security.prototype.getSystemPermissions = function ($happn, params, callback) {
-
-  try {
```

Comment:
does below not have the possiblity to throw anymore?

Reply:
If there is an error it means it is a bug, should never throw.